src/SFA.DAS.EmployerIncentives.Api/Controllers/EligibleApprenticeshipsQueryController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerIncentives.Abstractions.DTOs;
+using SFA.DAS.EmployerIncentives.Abstractions.Queries;
+using SFA.DAS.EmployerIncentives.Queries.NewApprenticeIncentive.GetApprenticeshipEligibility;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    [Route("eligible-apprenticeships")]
+    [ApiController]
+    public class EligibleApprenticeshipsQueryController : ApiQueryControllerBase
+    {
+        public EligibleApprenticeshipsQueryController(IQueryDispatcher queryDispatcher) : base(queryDispatcher)
+        {
+        }
+
+        [HttpGet("{uln}")]
+        public async Task<IActionResult> IsApprenticeshipEligible(long uln, [FromQuery]DateTime startDate, [FromQuery]bool isApproved)
```

Comment:
My understanding is you may also need the expectedDate date, stop date, pause date and date of birth to determine if they are eligible

Reply:
They're not part of this story so we can add them later on.